shared_model/schema/qry_responses.proto

```diff
@@ -37,6 +37,10 @@ message AccountAsset {
 // *** Responses *** //
 message AccountAssetResponse {
   repeated AccountAsset account_assets = 1;
+  uint32 total_number = 2;
+  oneof opt_next_asset_id {
```

Comment:
Why we don't use `optional ` modifier?

Reply:
AFAIK, `optional` modifier is a [proto2](https://developers.google.com/protocol-buffers/docs/proto#optional) feature, while we use [proto3](https://developers.google.com/protocol-buffers/docs/proto3) that does not have it.